bin/package

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+VERSION=${REF#"refs/tags/"}
+SRC=`pwd`
+DIST=$SRC/dist
+BIN=agora
+
+echo "Packaging $BIN $VERSION for $TARGET..."
+
+test -f Cargo.lock || cargo generate-lockfile
+
+echo "Building $BIN..."
+
+case $OS in
+  ubuntu-latest | macos-latest)
+    cargo rustc --bin $BIN --target $TARGET --release
```

Comment:
I think that `rustc` is not necessary here, but is necessary in the windows invocation, because we're passing extra arguments. I'll switch to `cargo build` here.